EntranceShuffle.py

```diff
@@ -2275,7 +3781,8 @@ def tuplize_lists_in_list(ls):
 # value = entrance #
 #        | (entrance #, exit #)
 exit_ids = {'Links House Exit': (0x01, 0x00),
-            'Chris Houlihan Room Exit': (None, 0x3D),
+            'Inverted Big Bomb Shop': 0x53,
+            'Chris Houlihan Room Exit': (0x82, 0x3D),
```

Comment:
In this case I was trying to add the non-inverted Pyramid Entrance back in to the shuffle and needed an extra cave.

Reply:
Yeah, since we pyramid exit to the light world, if we were to have something shuffled there, we would need an extra cave. If i were doing that though, I would likely consider adding say an extra instance of the mutliply used fairy fountain or dark world shop. It would be a bit less jarring than entering a cave and finding yourself in the middle of the Houlihan room.

As far as adding that as an extra entrance we would need to add an entry to the overworld doors table to handle it. Technically there is one entry in that that that is unused (it is a duplicate) so it would be possible. But is it worth it for just adding a fairly useless location, since it is not reachable, except by exiting from it?

Reply:
The old Pyramid Entrance can be mirrored to from HC ledge in inverted, on the lower left near where it widens out. But yeah, it's not super important to have especially since access to HC ledge is limited as well.